src/data/fintype/basic.lean

```diff
@@ -663,13 +670,52 @@ by simp [finset.subset_iff, set.subset_def]
   s.to_finset ⊂ t.to_finset ↔ s ⊂ t :=
 by simp only [finset.ssubset_def, to_finset_mono, ssubset_def]
 
+lemma subset_iff_to_finset_subset (s t : set α) [fintype s] [fintype t] :
+  s ⊆ t ↔ s.to_finset ⊆ t.to_finset :=
+by simp only [to_finset_mono]
```

Comment:
Is this just to_finset_mono in reverse? If so, why do we need this one?

Reply:
I think I only left it in because I didn't want to track down its uses -- but removing it wasn't so bad. Thanks.